waymo2argo/waymo_raw_data_to_argoverse.py

```diff
@@ -285,12 +285,12 @@ def dump_pose(city_SE3_egovehicle: np.ndarray, timestamp: int, log_id: str, pare
         log_id: Log ID that the reading belongs to
         parent_path: The directory that the converted data is written to
     """
-    x, y, z = city_SE3_egovehicle[:3, 3]
+    x1, y1, z1 = city_SE3_egovehicle[:3, 3]
     R = city_SE3_egovehicle[:3, :3]
     assert np.allclose(city_SE3_egovehicle[3], np.array([0, 0, 0, 1]))
     q = rotmat2quat(R)
     w, x, y, z = q
-    pose_dict = {"rotation": [w, x, y, z], "translation": [x, y, z]}
+    pose_dict = {"rotation": [w, x, y, z], "translation": [x1, y1, z1]}
```

Comment:
mqny thanks for catching this. I propose we make the following change instead
```python
    qw, qx, qy, qz = q
    pose_dict = {"rotation": [qw, qx, qy, qz], "translation": [x, y, z]}
```
Could you make that update @ruizhaoz and I'll merge it?